src/Microsoft.IdentityModel.Protocols/LogMessages.cs

```diff
@@ -40,8 +40,8 @@ internal static class LogMessages
         internal const string IDX20000 = "IDX20000: The parameter '{0}' cannot be a 'null' or an empty object.";
 
         // properties, configuration 
-        internal const string IDX20106 = "IDX20106: When setting RefreshInterval, the value must be greater than MinimumRefreshInterval: '{0}'. value: '{1}'.";
-        internal const string IDX20107 = "IDX20107: When setting AutomaticRefreshInterval, the value must be greater than MinimumAutomaticRefreshInterval: '{0}'. value: '{1}'.";
+        // internal const string IDX20106 = "";
```

Comment:
Messages removed as these exceptions are now being thrown from the Microsoft.IdentityModel.Tokens namespace.